src/middlewares/errorHandler.ts

```diff
@@ -5,12 +5,15 @@ import logger from '../lib/logger'
 export default function errorHandler (error: BaseError, req: Request, res: Response, next: NextFunction): void {
   const status = error.status || 500
   const message = error.message || 'Something went wrong '
+  const meta = error.meta || {}
 
   logger.error('Error Message:', {
```

Comment:
You need to reuse error message as a logged message, also log meta directly and save a stack trace for debug purposes.

```
  logger.error(error.message, {
    ...meta,
    stack: error.stack
  })
```

Reply:
fixed d05ad7787eac1bd92d123586f034466887fdb451